lib/src/GlobalFilters.cc

```diff
@@ -0,0 +1,86 @@
+#include <drogon/plugins/GlobalFilters.h>
+#include <drogon/DrClassMap.h>
+#include <drogon/HttpAppFramework.h>
+#include "FiltersFunction.h"
+#include "HttpRequestImpl.h"
+
+using namespace drogon::plugin;
+
+void GlobalFilters::initAndStart(const Json::Value &config)
+{
+    if (config.isMember("filters") && config["filters"].isArray())
+    {
+        auto &filters = config["filters"];
+
+        for (auto const &filter : filters)
+        {
+            if (filter.isString())
+            {
+                auto filterPtr = std::dynamic_pointer_cast<HttpFilterBase>(
+                    drogon::DrClassMap::getSingleInstance(filter.asString()));
+                if (filterPtr)
+                {
+                    filters_.push_back(filterPtr);
+                }
+                else
+                {
+                    LOG_ERROR << "Filter " << filter.asString()
+                              << " not found!";
+                }
+            }
+        }
+    }
+    if (config.isMember("exempt"))
+    {
+        auto exempt = config["exempt"];
+        if (exempt.isArray())
```

Comment:
I think we should add some error handling here. It's frustrating when config does nothing and the framework generates no diagnostic.